### DISCUSSION TITLE
Συμπληρωματική συνάντηση ΟΣΣ 3

### DISCUSSION THREAD

**Discussion by Katethoma** [sdy60/syllabus#58] [Q&A] [answered]
Καλησπέρα . Χρόνια Πολλά και Καλή Χρονιά ! 
Ήθελα να ρωτήσω πότε θα κάνουμε μια συνάντηση ΟΣΣ εντός του Ιανουαρίου για να λύσουμε κάποιες απορίες για την ΓΕ3 ?

**Comment by epidrome** [comment, marked as answer]
ναι φυσικά εγώ είμαι διαθέσιμος αυτό το ΣΚ στις 6μμ. 

αν θέλετε μπορείτε να ανοίξετε μια ψηφοφορία ή αν έχετε ήδη συντονιστεί με τους συναδέλφους να πούμε για αύριο Σάββατο στις 6μμ;

επίσης πάντα μπορείτε να κάνετε ερωτήσεις σε αυτήν την περιοχή (που παραμένει κάπως ήσυχη)

ή μπορείτε να κανονίσετε ατομική τηλ συνάντηση μετά από μεηλ.

καλή χρονιά!

**Comment by Katethoma** [comment]
Σας ευχαριστούμε κε Χωριανόπουλε ! Συμφωνούμε με την ενδιάμεση ΟΣΣ για αύριο Σάββατο στις 6.
Καλό βράδυ

________________________________
Από: Konstantinos Chorianopoulos ***@***.***>
Στάλθηκε: Friday, January 20, 2023 5:11:12 PM
Προς: sdy60/syllabus ***@***.***>
Κοιν.: ΘΩΜΑ ΑΙΚΑΤΕΡΙΝΗ ***@***.***>; Author ***@***.***>
Θέμα: Re: [sdy60/syllabus] Ερώτημα για συμπληρωματική συνάντηση ΟΣΣ (Discussion #58)


ναι φυσικά εγώ είμαι διαθέσιμος αυτό το ΣΚ στις 6μμ.

αν θέλετε μπορείτε να ανοίξετε ένα δημοψήφισμα ή αν έχετε ήδη συντονιστεί με τους συναδέλφους να πούμε για αύριο Σάββατο στις 6μμ;

επίσης πάντα μπορείτε να κάνετε ερωτήσεις σε αυτήν την περιοχή (που παραμένει κάπως ήσυχη)

ή μπορείτε να κανονίσετε ατομική τηλ συνάντηση μετά από μεηλ.

καλή χρονιά!
